### DISCUSSION TITLE
I created a crate that lets you draw petgraphs in jupyter

### DISCUSSION THREAD

**Discussion by timthelion** [petgraph/petgraph#689] [Show and tell]
Here is an example:

https://github.com/timthelion/petgraph-evcxr/blob/master/README.ipynb

**Comment by XVilka** [comment]
Thank you!

**Comment by timthelion** [comment]
Should I close this issue now? Or do you want to document it somewhere?

**Reply by XVilka** [reply]
I converted this into discussion.
